gamestonk_terminal/forex/fx_view.py

```diff
@@ -10,13 +10,28 @@
 import matplotlib.ticker as mticker
 import seaborn as sns
 
+# pylint: disable=R0402
 import oandapyV20.endpoints.pricing as pricing
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.accounts as accounts
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.orders as orders
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.instruments as instruments
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.positions as positions
+
```

Comment:
can this not just be `from oandapyV20.endpoints import (pricing,accounts,...)`

Reply:
I was just getting my pylint to work. I didn't add this file.

Reply:
Can you do @jmaslek suggestion, so on the top do

```
from oandapyV20.endpoints import (
   pricing,
   accounts,
   orders,
   ...
)
```
and then you don't need the `pylint: disable=R0402` anymore